src/kirby/components/card/card.component.tns.ts

```diff
@@ -8,7 +8,7 @@ import { registerElement } from 'nativescript-angular';
 import { ContentView } from 'tns-core-modules/ui/content-view';
 
 import { ScssHelper } from '../../scss/scss-helper';
-import { ColorType, ColorHelper } from '../../helpers/color-helper';
+import { ColorType } from '~/kirby/helpers/color-type';
```

Comment:
Don't use `~` imports as it breaks symlinking (there's as reason why the line above uses relative path...)